remoteworkitem/flatten_whitebox_test.go

```diff
@@ -0,0 +1,51 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFlattenMapWhitebox(t *testing.T) {
```

Comment:
Do you test anything new in this function that hasn't been tested with `TestFlattenMap` in `remoteworkitem/flatten_test.go`?

Also, there's no need to call this `TestFlattenMapWhitebox` the other `TestFlattenMap` lives in a different package (`remoteworkitem_test`).